### PR TITLE
Add ability to preceeed external shell commands with '.'. Add timings for exach step. Rename prompt to 'gpt' from 'engshell'.

### PR DESCRIPTION
**Pull request by cschladetsch**

### CODE REVIEW

File: bubble_sort.cpp
Comment (emcf) [inline bubble_sort.cpp:1]: shouldn't commit this

File: .gitignore
Comment (emcf) [inline .gitignore:1]: good idea

File: README.md
Comment (emcf) [inline README.md:1]: Good minor changes. You seem to have a very specific use case that is concerned with execution times. An optional --time would be better. Also, no need to change the name of the project at this time nor change the prompt to gpt until we have compatibility with other LLMs

File: engshell.py
Comment (emcf) [inline engshell.py:1]: Great catch on the prompt length fix.
Unsplash is optional, we want to be minimal with requirements and allow options from there so no need to import the key.
Perhaps `from keys import *` could work well here?

File: game.html
Comment (emcf) [inline game.html:1]: Don't need to commit this

File: prompts.py
Comment (emcf) [inline prompts.py:1]: Good idea to allow different python versions by changing pip3 to pip. We could go one step further even and do
```
INSTALL_USER_MESSAGE = lambda package: f"""Write the {OPERATING_SYSTEM} pip command for python {PYTHON_VERSION} so I can install {package}. Please do not explain, return only the single command to install it."""
```

**Comment by cschladetsch** [comment]
Yes, sorry it was late at night. I'll review the PR tomorrow and try again with fewer overt changes.

**Pull request closed without merging**